connectivity/service/src/main/java/org/eclipse/ditto/connectivity/service/messaging/httppush/AsyncJwtLoader.java

```diff
@@ -147,21 +148,34 @@ private DittoRuntimeExceptionBuilder<JwtInvalidException> getJwtInvalidException
     private static HttpRequest toTokenRequest(final String tokenEndpoint,
             final String clientId,
             final String clientSecret,
-            final String scope) {
+            final String scope,
+            final Optional<String> audience) {
```

Comment:
We don't like passing `Optional`s as argument to methods.
Could you please convert that instead to:
```suggestion
            @Nullable final String audience) {
```

Reply:
updated it